docs/sources/k6/next/using-k6-browser/playwright-apis-in-k6.md

```diff
@@ -0,0 +1,457 @@
+---
+title: 'Playwright APIs in k6'
+description: 'A table of Playwright APIs which are implemented in k6'
+weight: 350
+---
+
+# Playwright APIs in k6
+
+The following is the key for what the statuses mean.
+
+| Status | Description |
+| ------ | ----------- |
+| 🟥 | Not implemented in k6 Browser |
+| 🟩 | Implemented in k6 Browser |
+| 🟧 | k6 Browser only |
+
+The following table shows which Playwright APIs are implemented in k6. We aim to match Playwright's API behavior and signature exactly, but some differences may exist. For more details, refer to the individual API documentation page.
```

Comment:
```suggestion
The following table shows which Playwright APIs are implemented in k6. k6 tries to match Playwright's API behavior and signature exactly, but some differences may exist. For more details, refer to the individual API documentation page.

The status column has three different options:

| Status | Description |
| ------ | ----------- |
| 🟩 | Implemented in k6 Browser |
| 🟥 | Not implemented in k6 Browser |
| 🟧 | k6 Browser only |
```

I think we can switch the order here to help the reader understand what this page is.